web_src/js/features/admin/selfcheck.js

```diff
@@ -0,0 +1,31 @@
+import {toggleElem} from '../../utils/dom.js';
+import {POST} from '../../modules/fetch.js';
+
+const {appSubUrl} = window.config;
+
+export async function initAdminSelfCheck() {
+  const elCheckByFrontend = document.querySelector('.page-content.admin .self-check-by-frontend');
+  if (!elCheckByFrontend) return;
+
+  const elContent = document.querySelector('.page-content.admin .admin-setting-content');
+
+  // send frontend self-check request
+  const resp = await POST(`${appSubUrl}/admin/self_check`, {
+    data: new URLSearchParams({
+      location_origin: window.location.origin,
+      now: Date.now(), // TODO: check time difference between server and client
```

Comment:
Why send this at all?

Reply:
This is a TODO for this, and I have said it is a TODO:

![image](https://github.com/go-gitea/gitea/assets/2114189/8f9dafc8-e6bd-48be-a314-d52ebbcdc645)